core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -60,8 +68,8 @@
   private final Lock mWriteLock;
 
   /** Maps from Alluxio path string, to {@link MountInfo}. */
```

Comment:
this comment should be moved to `State`, and this line should be have a different comment.

Reply:
fixed, thanks